ee/indexeddb/indexeddb.go

```diff
@@ -0,0 +1,184 @@
+// Package indexeddb provides the ability to query an IndexedDB and parse the objects it contains.
+package indexeddb
+
+import (
+	"bytes"
+	"encoding/binary"
+	"errors"
+	"fmt"
+	"io"
+	"os"
+	"path/filepath"
+
+	"github.com/kolide/goleveldb/leveldb"
+	"github.com/kolide/goleveldb/leveldb/opt"
+	"github.com/kolide/launcher/ee/agent"
+)
+
+// maxObjectStoresToCheck is the number of indices for object stores we will check
+// before declaring failure to find the given object store. We cannot look up
+// object stores by their names, only by their IDs -- so we have to iterate through
+// up to maxObjectStoresToCheck IDs to find the desired store. We assume there are
+// fewer than 100 object stores in a given database.
```

Comment:
Do we know enough to make this a good guess? (We might need more data from the field)

Reply:
The extension that we care about currently has 10 object stores, so 100 felt like a safe number for this particular usecase (plenty of wiggle room if this extension starts to add more object stores).

My guess is that it would be weird/unusual for an extension to have more than 100 object stores, but that could be incorrect.

Reply:
Updated the comment to note that we may need to adjust this number upward in the future